src/main/java/io/ipfs/api/IPFS.java

```diff
@@ -490,6 +492,28 @@ public Object log() throws IOException {
         }
     }
 
+    private boolean versionIsValid(String version) {
+        String[] parts = version.split("\\.");
```

Comment:
@kubuxu I prefer to avoid bringing in binary external dependencies for simple things like this. One major benefit of that approach is that it makes it easy to cross compile this library to JS (which I do elsewhere).